### PR TITLE
Wm7 add project members

### PR DESCRIPTION
**Pull request by Pavel-An**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+  
+  private 
+  
+  def find_project
+    @project = Project.find(params[:id])
+  end
+
+  def find_project_by_id
+    @project = Project.find(params[:project_id])
+  end
```
Comment (fooooxmr) [inline app/controllers/application_controller.rb:19]: This looks strange. Why sometimes you are receiving id as id, and sometimes as project_id?

Comment (fooooxmr) [reply app/controllers/application_controller.rb:19]: And why this code inside the ApplicationController?

File: app/controllers/members_controller.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  def edit
+    
+  end
+
+  def update
+  end
+
+  def destroy
+    @member.destroy if @project.members.count > 1
```
Comment (fooooxmr) [inline app/controllers/members_controller.rb:30]: I guess, we should show some message to the user, if destroy was successfull  or not.

File: app/controllers/members_controller.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    @members = @project.members
+  end
+
+  def new
+    @members = User.where.not(id: @project.members.pluck(:user_id).compact)
+  end
+
+  def create
+    members_params[:user_id].each do |id|
+      @members = @project.members.create(user_id: id)
```
[the remark this answers is not in the document]
Comment (fooooxmr) [reply app/controllers/members_controller.rb:16]: Also, what will be if user already member of this project? Will you assign this user one more time?

**Review by fooooxmr — CHANGES_REQUESTED** [review]
See comments.

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (Pavel-An) [reply app/controllers/application_controller.rb]: done

File: app/controllers/members_controller.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (Pavel-An) [reply app/controllers/members_controller.rb]: done

File: app/controllers/members_controller.rb (hunk 2 above)
Comment (Pavel-An) [reply app/controllers/members_controller.rb:35]: done

**Pull request merged**